### ISSUE TITLE
Task Assignee Reward Crash

### ISSUE THREAD

**Issue by pavlovcik** [ubiquity/ubiquity-dollar#782]
> Error: Error: call revert exception [ See: https://links.ethers.org/v5-errors-CALL_EXCEPTION ] (method="symbol()", data="0x", errorArgs=null, errorName=null, errorSignature=null, reason=null, code=CALL_EXCEPTION, version=abi/5.7.0)

_Originally posted by @ubiquibot[bot] in https://github.com/ubiquity/ubiquity-dollar/issues/663#issuecomment-1696985103_

**Comment by molecula451** [comment]
nope, the bug here, is the front end is calling a method .symbol() in the contract to render like <WXDAI> etc but it's not available. But why is on this repo?

**Comment by molecula451** [comment]
nope, the bug here, is the front end is calling a method `.symbol()` in the contract to render like etc but it's not available. But why is on this repo?
